tests/stubtest_whitelists/py39.txt

```diff
@@ -34,8 +34,6 @@ collections.abc.ItemsView.__reversed__
 collections.abc.KeysView.__reversed__
 collections.abc.ValuesView.__reversed__
 # pending merge of https://github.com/python/cpython/pull/22023
```

Comment:
should this comment have been removed too?

Reply:
Yup, good catch! I had meant to keep an eye out for that...  #4603